Testing/VelaptorTests/UI/LabelTests.cs

```diff
@@ -177,18 +179,18 @@ public void WidthAndHeight_WhenTextIsEmpty_CalculatesCorrectResults()
             Assert.Equal(0u, actualHeight);
         }
 
-        [Fact]
+        [Fact(Skip = "Cannot be tested until IFont is injected for mocking.")]
         public void Style_WhenSettingValue_ReturnsCorrectResult()
```

Comment:
FYI - This is actually a property name, not a method name.  There are no parenthesis.  Also, the property that is being tested is on line 190, not on 189.  

VelFontStyle is an alias because there is a naming conflict between 2 types across the namespaces that are being used in this file. 

VelFontStyle is an alias for FontStyle and FontStyle is an enumeration which is technically a data type.